staging/src/k8s.io/kubectl/pkg/cmd/delete/delete.go

```diff
@@ -272,6 +276,10 @@ func (o *DeleteOptions) Validate() error {
 		}
 	}
 
+	if o.warningPrinter == nil {
```

Comment:
This check needs to be higher or it will panic above.

Reply:
done